src/main/java/com/avioconsulting/log4j/sqs/processor/FifoMessageProcessor.java

```diff
@@ -21,55 +21,54 @@
  */
 public class FifoMessageProcessor implements LogEventProcessor {
 
-	private static final Logger logger = LogManager.getLogger(FifoMessageProcessor.class);
+    private static final Logger logger = LogManager.getLogger(FifoMessageProcessor.class);
 
-	@Override public MessageRequestWrapper process(ProcessorAttributes processorAttributes) {
-		List<SendMessageRequest> sendMessageRequestList = new ArrayList<>();
-		logger.debug("Fifo - Splitting large message");
-		// Generate Message Hash to use as the group ID
-		UUID uuid = UUID.randomUUID();
-		logger.debug("Large Message UUID: {}", uuid);
-		String[] splitMessage = splitStringByByteLength(processorAttributes.getMessage(), "UTF-8", processorAttributes.getMaxMessageSize());
-		MessageRequestWrapper messageRequestWrapper = new MessageRequestWrapper();
+    @Override
+    public MessageRequestWrapper process(ProcessorAttributes processorAttributes) {
+        List<SendMessageRequest> sendMessageRequestList = new ArrayList<>();
+        logger.debug("Splitting large message");
+        UUID uuid = UUID.randomUUID();
+        logger.debug("Large Message UUID: {}", uuid);
+        String[] splitMessage = splitStringByByteLength(processorAttributes.getMessage(), "UTF-8", processorAttributes.getMaxMessageSize());
+        MessageRequestWrapper messageRequestWrapper = new MessageRequestWrapper();
 
-		for (int i = 0; i < splitMessage.length; i++) {
-			logger.debug("Sending message {} of {}", i + 1, splitMessage.length);
-			SendMessageRequest sendMessageRequest = new SendMessageRequest();
-			sendMessageRequest.setMessageBody(String.format("currentPart=%d|totalParts=%d|uuid=%s|message=%s", i + 1, splitMessage.length, uuid, splitMessage[i]));
-			sendMessageRequest.setMessageGroupId(uuid.toString());
-			sendMessageRequest.setQueueUrl(processorAttributes.getQueueUrl());
-			sendMessageRequestList.add(sendMessageRequest);
-			messageRequestWrapper.setSendMessageRequest(sendMessageRequestList);
-		}
-		return messageRequestWrapper;
-	}
+        for (int i = 0; i < splitMessage.length; i++) {
+            logger.debug("Sending message {} of {}", i + 1, splitMessage.length);
```

Comment:
We should make sure to include the UUID here if we're going to log each part, otherwise anything more than a single request at a time is going to get hard to follow

Reply:
That is  just an informative log of the appender, not the actual message to be sent to sqs. If you are agree, I can add the uuid at here.

Reply:
I think it would still be good to log the UUID so you can tell in the appender logs that all of the log messages are for the same set of messages to SQS